app/models/patient.rb

```diff
@@ -188,7 +188,7 @@ def high_risk?
 
   def current_age
     if date_of_birth.present?
-      Date.current.year - date_of_birth.year
+      ((Time.zone.now - date_of_birth.to_time) / 1.year).floor
```

Comment:
I remember looking at the current age method before I even started working at simple and thinking, "That doesn't seem right. You can't know someone's age just be subtracting years."

Reply:
Intuitively I felt there should be a better solution to get an interval's value in years. But apparently this is the best we can do 😁 . Glad we got this fixed.